linux-install/README.md

```diff
@@ -0,0 +1,30 @@
+# RLBotGUI Ubuntu (Linux) Install
+
+## First-time installation
+
+There are a few steps to install this on Linux.
+
+1. Download `RLBotGUI.sh` (Or save [this file](https://raw.githubusercontent.com/RLBot/RLBotGUI/master/linux-install/RLBotGUI.sh))
+2. `chmod +x /path/to/RLBotGUI.sh`
+3. `path/to/RLBotGUI`
+4. You will be asked to provide your password. The RLBotGUI requires sudo in order to run:
+
+   - `sudo apt-get update`
+   - `sudo apt-get install python3.8` (If needed)
+   - `sudo apt-get install python3.8-venv` (If needed)
```

Comment:
We've standardized on python 3.7 in windows world, and we may want to do the same for Linux. 
- If RLU gets compiled for Linux, it could still only target 3.7 as it does in Windows
- There may be other projects that already work on Linux relying on the 3.7 assumption (but I'm not aware of any)

Reply:
Ubuntu 20.04 comes pre-installed with Python 3.8, and Python 3.8 gets auto-update with Ubuntu. This is why I chose Python 3.8 over 3.7. Do you still want me to change to Python 3.7?

Reply:
Ah, I didn't know that.

One other factor I forgot is that all our tournaments are run on python 3.7 and that will probably not change for the foreseeable future. Therefore if somebody develops a python bot and uses features from 3.8, their bot will not work in the tournament. That would be a headache for the bot maker and also the tournament organizer.

For that reason I think I still prefer 3.7.

Reply:
Very well.